code/src/index.css

```diff
@@ -1,13 +1,67 @@
-body {
+:root {
+  --leftMargin: calc(5% + 130px);
+  --sidebarSmallWidth: calc(130px / 3);
+  --lightgrey: #a0a0a0;
+  --darkBackground: rgb(27, 27, 27);
+  --overlayColor: #0f1011a6;
+}
+
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+}
+body {
+  background: #202124;
+  width: 100vw;
+  overflow-x: hidden;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+/* General class to use when an element is toggled from display:none */
+.display {
+  display: block !important;
+}
+
+/* ------------------------------- TYPOGRAPHY ------------------------------- */
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.link,
+.text-inline {
+  font-size: 14px;
+  line-height: 1.5em;
+  font-weight: normal;
+  text-decoration: none;
+}
+.text--small {
+  font-size: 0.8em;
+  text-transform: uppercase;
+}
+.list__title {
```

Comment:
why `__` instead of `--`?? when you are using the BEM you should use `--` to point the element and `__` the state or action. 

`navigation--item__active`

Reply:
Oh really?! I thought it was the other way around 🙈 
I will take note if this 😉